src/serialization/sb2.js

```diff
@@ -251,10 +251,14 @@ const parseScratchObject = function (object, runtime, extensions, topLevel) {
         for (let k = 0; k < object.lists.length; k++) {
             const list = object.lists[k];
             // @todo: monitor properties.
-            target.lists[list.listName] = new List(
+            const newVariable = new Variable(
+                getVariableId(list.listName),
                 list.listName,
-                list.contents
+                Variable.LIST_TYPE,
+                list.isPersistent
```

Comment:
@rschamp can you confirm that lists are never persistent (cloud) coming from SB2? If so, this should just be changed to `false`, or just left out and forced to `false` for all list types?

Reply:
Correct, cloud variables are only scalar.